src/sherpa_ai/tools.py

```diff
@@ -202,3 +211,43 @@ def _run(self, query: str) -> str:
 
     def _arun(self, query: str) -> str:
         raise NotImplementedError("UserInputTool does not support async run")
+
+class HugChatTool(BaseTool):
+    name = "Hugchat"
+    description = (
+        "Access the user input for the task."
+        "You use this tool if you need to use HugChat for Q&A."
```

Comment:
I think it will help everyone if you explain why someone might want to use HugChat for Q&A, versus other alternatives. Maybe something like this... "This tool is an alternative way to to ask clarifying questions to solve the task, using HuggingChat via the HugChat API"  ?